content/influxdb/cloud-dedicated/reference/internals/storage-engine.md

```diff
@@ -38,11 +40,29 @@ queries, and is optimized to reduce storage cost.
 
 ## Storage engine components
 
+- [Router](#router)
 - [Ingester](#ingester)
 - [Querier](#querier)
 - [Catalog](#catalog)
 - [Object store](#object-store)
 - [Compactor](#compactor)
+- [Garbage collector](#garbage-collector)
+
+### Router
+
+The Router (also known as the Ingest Router) parses and shards incoming line
```

Comment:
There's no sharding. The entire line protocol request (HTTP POST body) is parsed, then passed to 2 (or 1 or 17; this is configurable) Ingesters via gRPC. Partitioning (but not sharding) happens when the Ingesters eventually write data files to object storage.